Automattic/wp-calypso
# Help-center: implement Sibyl suggestions

#### Changes proposed in this Pull Request

<img src="https://user-images.githubusercontent.com/17054134/167851672-57ac221f-c363-4dd6-ba1b-aed640228252.png" width="300">

#### Testing instructions

1. Run `yarn dev ---sync` in calypso/apps/editing-toolkit.
2. Sandbox a site and go to the contact form in the help-center.
3. Type some message like `domain`.
4. You should see relevant suggestions. 
5. Search should be debounced by 500ms.

Context: p1652192901233259/1652191736.493179-slack-C02T4NVL4JJ
